### DISCUSSION TITLE
Blog post

### DISCUSSION THREAD

**Discussion by asad-awadia** [bbernhard/signal-cli-rest-api#358] [Show and tell]
Recently had my configs reset and I had to go through the entire setup process all over again.

The captcha stuff can be quite confusing so I wrote a post that can serve as a tutorial/documentation for other future users

I would appreciate if this were linked somewhere in the readme if it is useful

Link => https://blog.aawadia.dev/2023/04/24/signal-api/

**Comment by bbernhard** [comment]
Many thanks for writing a blog post - very much appreciated! I've added the link to the Readme :)
